src/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -24,6 +24,13 @@ class FunctionCommentSniff extends PEARFunctionCommentSniff
      */
     private $phpVersion = null;
 
+    /**
+     * Whether to use short forms of type keywords.
+     *
+     * @var boolean
+     */
+    public $useShortTypes = false;
```

Comment:
Shouldn't this default to true due to PSR-12?

Reply:
Please don't request changes which are in violation with the coding standard used by the library.

Reply:
> Shouldn't this default to true due to PSR-12?

Problem is that this might be considered as a breaking change. I'm not sure how this was done in the past, but maybe it should remain `false` in PHPCS 3.x and changed to `true` in 4.x.

Reply:
I understand. In the mean time it's easy to just change that property in the config file.